site/about/glossary/key_concepts/_metrics.qmd

```diff
@@ -0,0 +1,4 @@
+metrics, custom metrics
+: Metrics are a subset of tests that do not have thresholds. Custom metrics are functions that you define to evaluate your model or dataset. These functions can be registered with ValidMind to be used in the platform.
+
+    -  In the context of ValidMind's Jupyter Notebooks, metrics and tests can be thought of as interchangeable concepts. 
```

Comment:
```suggestion
    -  In the context of ValidMind's Jupyter notebooks, metrics and tests can be thought of as interchangeable concepts. 
```

AFAIK, that's the correct spelling "Jupyter notebook", as in the notebooks themselves. But there is also "Jupyter Notebook", as in the web-based platform (which this isn't a reference to).